src/app/content/utils/urlUtils.ts

```diff
@@ -80,15 +80,15 @@ const getCommonParts = (firstPath: string[], secondPath: string[]) => {
   return result;
 };
 
-const trimTrailingSlash = (path: string) => path.replace(/([^/]{1})\/+$/, '$1');
-
 export const toRelativeUrl = (from: string, to: string) => {
-  const parsedFrom = trimTrailingSlash(from).split('/');
-  const parsedTo = trimTrailingSlash(to).split('/');
+  const parsedFrom = from.split('/');
+  const parsedTo = to.replace(/\/$/, '').split('/');
```

Comment:
i don't think we want to be trimming the trailing slashes off of anything. if you leave this as it is passed in, and change all the logic below back to the way it started, i believe everything works the way we need it to

Reply:
Original logic assumed that the last part of `parsedTo` will be actually a part of url not an empty string. Because of that, there would need to be a check to how many parts to subtract or the part that repeats '../' would need to be adjusted

Reply:
That is of course if there always should be a part of the actual url returned. I assumed that links like '../' are incorrect